service/service.go

```diff
@@ -68,6 +70,15 @@ func New(config Config) (*Service, error) {
 
 	var err error
 
+	var azureConfig *azure.Config
+	{
+		azureConfig := azure.DefaultConfig()
+		azureConfig.ClientID = config.Viper.GetString(config.Flag.Service.Azure.ClientID)
+		azureConfig.ClientSecret = config.Viper.GetString(config.Flag.Service.Azure.ClientSecret)
+		azureConfig.SubscriptionID = config.Viper.GetString(config.Flag.Service.Azure.SubscriptionID)
+		azureConfig.TenantID = config.Viper.GetString(config.Flag.Service.Azure.TenantID)
+	}
```

Comment:
The dependencies were wrong the AzureConfig is needed for the operator service but not the main service. This is fixed now.